WcaOnRails/app/views/regulations/translations/index.html.erb

```diff
@@ -10,11 +10,31 @@
 
   <% current_translations = [
     {
-      version: "2020-01-01",
+      version: "2020-08-01",
       language: "简体中文",
       language_english: "Chinese",
       url: "./chinese",
     },
+    {
+      version: "2020-08-01",
+      language: "Suomi",
+      language_english: "Finnish",
+      url: "./finnish",
+    },
+    {
+      version: "2020-08-01",
+      language: "Português Brasileiro",
+      language_english: "Portuguese, Brazil",
+      url: "./portuguese-brazilian",
+    },
+  ] %>
+
+  <h3><%= t("regulations_translations.current") %></h3>
+  <div class="regulations-translations-table-container">
+    <%= render 'regulations/translations/index_table', translations: current_translations %>
+  </div>
+
+  <% old_translations = [
     {
```

Comment:
I believe the Kazakh translation should be between the russian and the japanese ones, right?

Although this was not originally observed when adding Kazakh, I believe we sort first by the most recent translations and then by alphabetical order in english.